src/react/databrowser/objects/ObjectList.jsx

```diff
@@ -2,8 +2,9 @@
 
 import * as L from '../../ui-elements/ListLayout2';
 import * as T from '../../ui-elements/Table';
+import type { BucketInfo, ListObjectsType, Object } from '../../../types/s3';
```

Comment:
I checked `types/s3` to see what `Object` is.

Is it recommended/intentional to use the reserved `Object` ?

Reply:
It was not intentional. ZenkoUI has a compiler to strip away Flow types.

Reply:
I will change it to ObjectEntity for readability.